src/Picasso_FieldTypes.hpp

```diff
@@ -321,6 +321,21 @@ struct ScalarViewWrapper
         return _v( i0, i1, 0 );
     }
 
+    // Access data through point-wise array-based index arguments.
+    template <int VR = view_rank>
+    KOKKOS_FORCEINLINE_FUNCTION std::enable_if_t<4 == VR, linear_algebra_type&>
+    operator()( const int i[3] ) const
+    {
```

Comment:
`VR` is only needed to make `view_rank` a dependent template parameter for SFINAE, so it's good practice to prevent users from specifying an inappropriate value with a `static_assert`, something like this:
```cpp
static_assert( 4 == VR, "This template parameter is for SFINAE only and should not be given explicitly");
```
I'll concede that the likelihood of a user doing this in the case of `operator()` is exceedingly small:
```cpp
vw = createViewWrapper(view);
vw.template operator()<3>( index );
```
No one is going to accidentally write that. I'm willing to let it slide in this case, but I'd argue it's still good practice. The same comment applies throughout.

Reply:
I agree with this - I have added these